_config.yml

```diff
@@ -1,8 +1,44 @@
 url: http://jsonapi.org
-
-lsi: false
 source: .
 destination: ./public
+
+# github pages we cannot customize
+lsi: false
+whitelist:
+  - jekyll-coffeescript
+  - jekyll-gist
+  - jekyll-paginate
+  - jekyll-redirect-from
+  - jekyll-mentions
+  - jekyll-sitemap
+  - jekyll-feed
+  - jekyll-seo-tag
+  - jekyll-gist
+  - jemoji
+incremental: false
+highlighter: rouge
+gist:
+  noscript: false
+# `safe `must be set false for jekyll-redirect-from
+# to run in development. (Github Pages will override
+# `safe` to be true on deploy, but it makes an
+# exception for jekyll-redirect from.)
+safe: false
+# customize github pages
+quiet: false
+kramdown:
+  input: GFM
+  hard_wrap: false
+  template: '' # cannot customize
+  math_engine: mathjax # cannot customize
+  syntax_highligher: rouge # cannot customize
+jailed: false
+gems:
+  - jekyll-redirect-from
+markdown: redcarpet
```

Comment:
originally I removed the markdown/redcarpet declarations, but then syntax highlighting was off and perhaps there were other formatting errors